oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSink.java

```diff
@@ -91,6 +91,12 @@ public static class OracleSinkConfig extends AbstractDBSpecificSinkConfig {
     @Nullable
     public Integer defaultBatchValue;
 
+    @Name(OracleConstants.ISOLATION_LEVEL)
+    @Description("The transaction isolation level for the database session.")
+    @Nullable
+    @Macro
+    public String transactionIsolationLevel;
```

Comment:
+1, we'd better add more descriptions here suggesting what transaction level to set for corresponding role. As I recall, you can not set arbitrary isolation level for sysdba or sysopr role, that's why we have https://github.com/data-integrations/database-plugins/blob/develop/oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java#L109

Reply:
also since isolation level is a common connection property , if we are only focusing on Oracle now, let's add a TODO to move it to AbstractDBSpecificConnectorConfig/AbstractDBConnectorConfig in the future

Reply:
@seanzhougoogle I kept the isolation level changes just to the Oracle Plugin. I'm currently working on the broader change across the other DB's and I'm using AbstractDBSpecificConnectorConfig/AbstractDBConnectorConfig for those

Reply:
@seanzhougoogle  I can confirm on the sysdba user failing w/ serialized isolation levels. I recieve ORA-08178: illegal SERIALIZABLE clause specified for user INTERNAL

We should document this in the docs (plugin) that any selection w/ an internal user like sysdba or sysop will result in the isolation level being changed to read_committed.